### PR TITLE
Use ruff-format instead of black

### PR DESCRIPTION
**Pull request by pavelzw**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -31,18 +31,6 @@ include = [
...
-    \.eggs
-  | \.git
-  | \.venv
-  | build
-  | dist
-)/
-'''
-
 [tool.ruff]
 line-length = 100
```
[the remark this answers is not in the document]
Comment (pavelzw) [reply pyproject.toml:35]: Yes, then the python diff would be empty. @0xbe7a any preferences on the line length? I don't have strong opinions on this.

[the remark this answers is not in the document]
Comment (0xbe7a) [reply pyproject.toml:35]: Me neither

[the remark this answers is not in the document]
Comment (pavelzw) [reply pyproject.toml:35]: image
the allmighty AI has spoken.

[the remark this answers is not in the document]
Comment (mohahf19) [reply pyproject.toml:35]: coin flips are so old school

**Pull request merged**